generator/src/pkg/model/input.go

```diff
@@ -92,9 +92,14 @@ type ClusterLatency struct {
 	Latency float64 `json:"latency"`
 }
 
+type Setting struct {
+	Logging bool `json:"logging"`
+}
+
 type FileConfig struct {
 	ClusterLatencies []ClusterLatency `json:"cluster_latencies"`
 	Services         []Service        `json:"services"`
+	Settings         Setting          `json:"settings,omitempty"`
```

Comment:
Yes, that's right.  We are going to pass this setting  in configmap for the microservices. In a microservice, we are going to parse it in the `run.sh` script using `jq`. We need the `logging` to be there, either with `true` value or `false` value.  So when we set an struct with `omitempty` if we pass nothing there, it will convert it to a `false` value.

But if we consider it without `omitempty` and our enduser doesn't pass this value in the input file, then our generator is looking for such thing which is not passed and raise an error and does not generate anything!

The fastest way for me to develop it, was to set it as omitempty. Do you think that we should focus more on this?